app/views/admin/orders/_order_details_table.html.erb

```diff
@@ -12,7 +12,8 @@
         Id: <%= order.id %> <br>
         Created: <%= order_presenter.created_at %><br>
         User:  <%= order.user.email %><br>
-        Status: <%= order.state %>
+        Status: <%= order.state %><br>
+        Payment status: <%= order.payment.state %>
```

Comment:
I think you can make a small improvement here, pls check this article:
https://medium.com/@gioch/design-patterns-law-of-demeter-with-rails-49a44a9689fe
Or alternatively  maybe it should go to the presenter

Reply:
Great article! I could feel something could be improved here.